packages/components/pie-button/test/component/pie-button.spec.ts

```diff
@@ -525,3 +570,110 @@ test.describe('Form Actions', () => {
         });
     });
 });
+
+test.describe('props', () => {
+    test.describe('isResponsive', () => {
+        test.describe('when not set', () => {
+            test('the button should not have the attribute', async ({ mount }) => {
+                const component = await mount(
+                    PieButton,
+                    {
+                        props,
+                        slots: {
+                            default: 'Click me!',
+                        },
+                    },
+                );
+
+                await expect(component.locator('button'))
+                    .not.toHaveAttribute('isResponsive', '');
```

Comment:
I suspect this test will pass if the attribute has a different value, as it is only checking that the attribute is not `''`. I think you could just have `.not.toHaveAttribute('isResponsive');` and that would be fine

Reply:
I also was expecting it to work like that, but will only be available once we upgrade Playwright.

<img width="701" alt="image" src="https://github.com/justeattakeaway/pie/assets/448801/70890cf5-f792-42f2-959e-9d404f719227">

Reply:
I have added comments to remind us to clean it up after the upgrade.

Reply:
No problems from my side! 🙌🏼